src/odemis/driver/lakeshore.py

```diff
@@ -502,9 +501,18 @@ def _parseMessage(self, msg):
             vals = msg[5:].split(',')
             self._heating = int(vals[1])
         # Query temperature
-        elif re.match(b'SRDG\?', msg):
-            self._sendAnswer(b"%f" % (self._temperature,))
-            self._temperature += random.randint(-5, 5)
+        elif re.match(b'KRDG\?', msg):
+            # send temperature with some noise
+            self._sendAnswer(b"+%.3f" % (self._temperature + random.random() / 1000,))
```

Comment:
`random.random() / 1000` -> `random.uniform(-0.1, 01)`.
(I think that 0.1° is quite small already compared to the hardware precision, so that should be enough).